dask/dataframe/io/io.py

```diff
@@ -700,7 +707,7 @@ def from_delayed(
     if divisions is None or divisions == "sorted":
         divs = [None] * (len(dfs) + 1)
```

Comment:
An alternative to the unnecessary conversion to `list` below could be to just type this as a `collections.abc.Collection`, or, more simply, `list | tuple`.

```suggestion
        divs: list | tuple = [None] * (len(dfs) + 1)
```